data/cy/census_individual_gb_wls.json

```diff
@@ -5527,7 +5527,7 @@
                                         "answers": [
                                             {
                                                 "id": "last-year-address-answer-building",
-                                                "label": "Enw neu rif y t\u0177",
+                                                "label": "Cod post",
```

Comment:
The o should be the character ô in the word 'Cod'.

Reply:
This should be noted on crowdin, but shouldn't block this PR going in .